rasa/shared/core/domain.py

```diff
@@ -661,28 +672,16 @@ def _initialize_forms(
             `FormAction` which is implemented in the Rasa SDK.
         """
         if isinstance(forms, dict):
```

Comment:
@wochinge 
In reply to [this](https://github.com/RasaHQ/rasa/pull/7931#discussion_r615950111)

If I do not check if `forms` is a `dict` then this test `pytest tests/shared/core/test_domain.py::test_valid_slot_mappings` fails for the same reason as above, because `forms` is an empty dict, `{}` in this test.